src/main/java/jmgomez/apipolicy/feignClient/PolicyClient.java

```diff
@@ -1,24 +1,26 @@
 package jmgomez.apipolicy.feignClient;
 
+import jmgomez.apipolicy.config.FeignClientConfig;
 import jmgomez.apipolicy.model.Accident;
 import jmgomez.apipolicy.model.dto.PolicyDto;
 import jmgomez.apipolicy.model.dto.PolicyDtoCov;
 import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 
-@FeignClient(name = "policyClient", url = "http://localhost:8081")
+@FeignClient(name = "policyClient", url = "http://localhost:8081", configuration = FeignClientConfig.class)
```

Comment:
It doesn't hurt but you are using this just to configure the logging level. That's a problem if you want to change it depending on the env you are deploying this, for example. So bad choice. Now it's hardcoded. It can be done through the config file. Change it please.